Automattic/jetpack
# CLI: Prompt for auto-changelogger on pre-push

#### Changes proposed in this Pull Request:

Prompts the user on pre-push to run the changelogger if it finds projects still need one. *

#### Jetpack product discussion


N/a

But two questions:

- is committing the changelog files a good idea, or should we just leave it for the dev to do?
- I'd love to cancel the push, but with a custom message asking you to commit/push again, but I can't seem to figure out a good way to do so. It always fails, but I don't necessarily want someone to miss that the changelog file was added and they can push again. 
#### Does this pull request change what data or activity we track or use?


No
#### Testing instructions:






* Make some changes to a project
* Commit and try to push them 
* pre-push hook should notice a project needs a changelog and prompt you to add the changelog files
* It should add and commit the newly added changelog files and then prompt you to push again.
